### PR TITLE
Nathan/add announcements to home

### LINKED ISSUE #166: Add Announcements section to the homepage
Ideally, this should be a section near (or directly at) the top of the Homepage that displays the most recent Announcement from the Announcements page.

### PR DESCRIPTION
**Pull request by nathanzzhang**
fixes #166

### CODE REVIEW

**Review by jodymlin — CHANGES_REQUESTED** [review]
Looks great to me! Some minor changes requested :)

Also, could you show me what sample data you used to test this? And just to double check, have we cross-checked with @maggieelli and @jakobreinwald to make sure the test data matches the actual data we'll use?

File: src/components/HomePage/HomeAnnouncements.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+HomeAnnouncement.propTypes = {
+	subject: PropTypes.string,
+	timestamp: PropTypes.string,
+	body: PropTypes.string
+};
+
+export default function HomeAnnouncements() {
+	const classes = useStyles();
+	const announcement = data[0];
+	const [dismissed, changeVisibility] = useState(0);
```
Comment (khxia) [inline src/components/HomePage/HomeAnnouncements.js:115]: Why is the default value for the `useState(0)` a number when we are using it as a boolean? Should this be `false`?

File: src/components/HomePage/HomeAnnouncements.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+		paddingTop: theme.spacing(1),
+		paddingBottom: theme.spacing(2),
+		borderColor: theme.palette.primary.light,
+		borderRadius: 16,
+		marginTop: theme.spacing(2),
+		marginBottom: theme.spacing(2),
+		backgroundColor: '#fbf2ff'
+	},
+	button: {
+		width: 'auto',
```
[the remark this answers is not in the document]
Comment (nathanzzhang) [reply src/components/HomePage/HomeAnnouncements.js:49]: It apparently does nothing LOL (it was copied from the banner "apply" button)

File: src/components/HomePage/HomeAnnouncements.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+		textAlign: 'center',
+		textDecoration: 'none',
+		fontSize: '14px',
+		justifyItems: 'center',
+		fontWeight: 400,
+		margin: '5px 0px',
+		padding: '2px 1em',
```
[the remark this answers is not in the document]
Comment (jodymlin) [reply src/components/HomePage/HomeAnnouncements.js:56]: Oh yes this is okay :) It just multiples whatever value you put in by `8px` i'm pretty sure

File: src/components/HomePage/HomeAnnouncements.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+	);
+}
+
+HomeAnnouncement.propTypes = {
+	subject: PropTypes.string,
+	timestamp: PropTypes.string,
+	body: PropTypes.string
+};
+
+export default function HomeAnnouncementBanner() {
```
Comment (jodymlin) [inline src/components/HomePage/HomeAnnouncements.js:108]: By convention, we generally name the file the same name as the name of the main component we are exporting

**Review by jodymlin — CHANGES_REQUESTED** [review]
minor name changes also feel free to remove the sample data 👍 

after that lgtm 😄

**Review by khxia — APPROVED** [review]
looks good to me!

**Review by jodymlin — APPROVED** [review]

**Pull request merged**